### PR TITLE
Add HW RAID profile for IPI BM

### PR DESCRIPTION
**Pull request by jadhaj**

### CODE REVIEW

**Comment by jadhaj** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-baremetal-ipi-ovn-ipv4-hwraid-p1-f14

**Comment by jadhaj** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-baremetal-ipi-ovn-ipv4-hwraid-p1-f14

**Comment by jadhaj** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-baremetal-ipi-ovn-ipv4-hwraid-p1-f14

**Comment by jadhaj** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-baremetal-ipi-ovn-ipv4-hwraid-p1-f14

**Comment by jadhaj** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-baremetal-ipi-ovn-ipv4-hwraid-p1-f14

**Comment by jadhaj** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-baremetal-ipi-ovn-ipv4-hwraid-p1-f14

**Comment by jadhaj** [comment]
Hi @aleskandro 
I want to test this PR with Dell servers that have 2 disks or more
I need to hit swrvers fro doc: https://docs.google.com/spreadsheets/d/1UzDAZZId_TL4kU4DHaApYuu2JidYwiN7e9jKMrNpHHk/edit#gid=392544484

Is it possible to reserve by Vendor ?

**Comment by aleskandro** [comment]
Hi Jad, yes, there is a variable to reserve by vendor in the reserve step.

File: ci-operator/step-registry/baremetal/lab/ipi/install/baremetal-lab-ipi-install-ref.yaml [hunk 1]
```diff
@@ -41,6 +41,10 @@ ref:
       scenarios yet.
   - name: DISCONNECTED
     default: "false"
+  - name: HW_RAID
+    default: "false"
+    documentation: |-
+      Set to tru to configre hardeare RAID on nodes
```
Comment (jhou1) [inline ci-operator/step-registry/baremetal/lab/ipi/install/baremetal-lab-ipi-install-ref.yaml:47]: `tru` -> `true`

File: ci-operator/step-registry/baremetal/lab/ipi/install/baremetal-lab-ipi-install-commands.sh [hunk 2]
```diff
@@ -162,6 +162,11 @@ do
   cp "${item}" "${INSTALL_DIR}/manifests/${manifest##manifest_}"
 done < <( find "${SHARED_DIR}" \( -name "manifest_*.yml" -o -name "manifest_*.yaml" \) -print0)
 
+## HW RAID configuration
```
Comment (jhou1) [inline ci-operator/step-registry/baremetal/lab/ipi/install/baremetal-lab-ipi-install-commands.sh:165]: Do we have to uncomment to reconcile the env var setting in `HW_RAID` in baremetal-lab-ipi-install-ref.yaml?

**Pull request closed without merging**